ko/sql/dblink.rst

```diff
@@ -651,7 +652,219 @@ DBLink을 사용하기 위해 연결할 CUBRID의 broker들 정보 파악 또는
 
     CREATE SERVER 구문의 PROPERTIES 항목에 연결 속성을 설정할 수 있다. 자세한 내용은 :doc:`/sql/query/select` 와 :doc:`/sql/schema/server_stmt` 을 참고한다.
 
+유의사항
+==============================================
+
+*   테이블 확장 형식(object@server)은 테이블, 뷰, 동의어 만 대상으로 사용할 수 있으며, 시리얼, 내장함수, 저장함수 등은 대상으로 사용할 수 없다. 예를 들어, 원격서버(server1)의 sp_func() 저장 함수는 sp_func@server1(arg1, ...) 형식으로 사용할 수 없다.
+
+*   그러므로, SELECT 질의에 사용된 모든 함수들(SYSDATE 포함한 내장 함수, 저장 함수)은 모두 로컬에서 동작한다.
+*   하지만, SELECT질의와 다르게 INSERT/UPDATE/DELETE/MERGE 질의에 사용된 모든 함수들은 모두 원격 서버에서 동작한다.
+*   SELECT 질의에서 원격서버의 함수를 사용해야 하는 경우, DBLINK 구문을 사용할 수 있다.
+*   INSERT/UPDATE/DELETE/MERGE 질의에서 사용한 함수들은 원격 서버에서 수행되기때문에 함수 사용시 주의해야 한다. (즉, CUBRID 내장 함수 사용시 원격 DBMS에서 해당 내장 함수가 없거나 사용법이 다를 수 있음을 주의해야 한다.)
+*   미지원 타입
+    dblink는 원격 DB의 ODBC 드라이버를 통하여 처리되며, 원격DB의 ODBC 드라이버에서 미지원하는 타입은 지원하지 않는다. 단, INSERT, UPDATE, DELETE, MERGE와 같은 DML 구문에서는 입력값만 원격으로 전달하므로 미지원타입도 처리가 가능하다.
+조회 시에는 원격 DB의 ODBC 드라이버를 거치게 되어 미지원 타입의 데이터 조회는 지원하지 않는다.

```

Comment:
How about merging the above statement as commented?

Reply:
Ok